### ISSUE TITLE
Why am I can not get the mAP results in papers?

### ISSUE THREAD

**Issue by yingliu0518** [open-mmlab/mmrotate#522]
I validated some models, but get a mAP result below 10% to papers in DOTA dataset.Them get almost 70% mAP.
I used ms_trainval.json as config to split train imgs, and took ss_val.json as config to split val imgs.
Is my way to handle dataset wrong?

**Comment by nijkah** [comment]
I'm not sure which table you are checking. In most cases, the performance table on DOTA dataset is computed with the test set.
You can get the score by submitting the inference file in the DOTA evaluation server https://captain-whu.github.io/DOTA/evaluation.html.

Can I ask you to check it?

In addition to this, note that the learning rate and batch size can largely affect the final performance.
